docker/core/http/docker-compose.yml

```diff
@@ -104,3 +104,34 @@ services:
           - IOTA_HTTP_PORT=7896 # The port used for device traffic over HTTP
           - IOTA_PROVIDER_URL=http://<Cloud_Public_IP>:4041 # URL for the IoT Agent to be deployed
           - IOTA_DEFAULT_RESOURCE=/iot/json
+
+    elasticsearch:
+        image: fogflow/elasticsearch:7.5.1
+        ports:
+          - "9200:9200"
+          - "9300:9300"
+        environment:
+          - discovery.type=single-node
+#          - network.host= "0.0.0.0"
+    metricbeat:
+        image: fogflow/metricbeat:7.6.0
+        user: root
+        volumes:
+          - ./metricbeat.docker.yml:/usr/share/metricbeat/metricbeat.yml
+          - /var/run/docker.sock:/var/run/docker.sock
+          - /sys/fs/cgroup:/hostfs/sys/fs/cgroup
+          - /proc:/hostfs/proc
+          - /:/hostfs
+        environment:
+          - output.elasticsearch.hosts= ["<Cloud_Public_IP>:9200"]
```

Comment:
check the docker policy is it required restart(restart: always) the container at start or exit like other  containers
refer https://stackoverflow.com/questions/41555884/docker-what-does-docker-run-restart-always-actually-do